docs/samplesheets/examples.md

```diff
@@ -5,79 +5,175 @@ description: Examples of advanced sample sheet creation techniques.
 
 # Sample sheet channel manipulation examples
 
-## Separate entries based on a condition
+Each item in the channel emitted by `.fromSamplesheet()` is a flat tuple, corresponding with each row of the samplesheet. Each item will be composed of a meta value (if present) and any additional elements from columns in the samplesheet, e.g.:
+
+```csv
+sample,fastq_1,fastq_2,bed
+sample1,fastq1.R1.fq.gz,fastq1.R2.fq.gz,sample1.bed
+sample2,fastq2.R1.fq.gz,fastq2.R2.fq.gz,
+```
+
+Might create the channel:
+
+```nextflow
+tuple val(meta), path(fastq_1), path(fastq_2), path(bed)
```

Comment:
I'm not entirely convinced this is the cleanest and most clear way of representing the channel structure, maybe something like `[meta, fastq_1.fastq, fastq_2.fastq, sample1.bed]` will be more clear for beginners?

Reply:
It was to emulate a filename, but yeah this is just a matter of representation. I'm not sure which way is the best or most clear to beginners. (although both should be fine I think)